src/state/notifications/actions.ts

```diff
@@ -250,3 +257,9 @@ export const checkSubscriptionFailure = (error: string): CheckSubscriptionFailur
   type: NotificationAction.CheckSubscriptionFailureAction,
   error,
 });
+
+export type SetErrorActionFunction = (error: string) => SetErrorAction;
+export const setError: SetErrorActionFunction = error => ({
+  type: NotificationAction.SetErrorAction,
+  error,
+});
```

Comment:
To keep consistency please use setError instead FailureActions in Saga since it is more generic and does the same thing

Reply:
we shouldn't use setError in sagas because when we will want to add some listener on failed action, for example, fail of subscription, we won't be able to do it, we have to know explicitly what action has failed